src/test/java/org/jenkinsci/plugins/visualworks_store/StoreSCMConfigurationTest.java

```diff
@@ -25,23 +25,21 @@
 package org.jenkinsci.plugins.visualworks_store;
 
 import hudson.model.FreeStyleProject;
+import org.junit.jupiter.api.Test;
 import org.jvnet.hudson.test.JenkinsRule;
 import org.junit.Rule;
-import org.junit.Test;
 
 import java.util.Arrays;
 import java.util.List;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
 
 public class StoreSCMConfigurationTest {
```

Comment:
Should be 
```java
@WithJenkins
class StoreSCMConfigurationTest {

@Test
void testGlobalConfigurationRoundtrip(JenkinsRule j) throws Exception {

}
````

See docs for `@WithJenkins` for other examples https://javadoc.jenkins.io/component/jenkins-test-harness/org/jvnet/hudson/test/junit/jupiter/WithJenkins.html

Reply:
Thanks! 👍

I'm sorry, but I still need your expertise. 😊 Is it because I forgot to apply one recipe, one recipe didn't work, or is it part of the manual changes we have to do after applying the recipes?

Reply:
No worries, glad I can help.

From what I can see the recipes worked all fine. The part that is missing here must be done manually as you already pointed out. As you can can see in the Javadoc I linked there are multiple ways of doing this. That’s why I prefer to take a closer look into those parts rather than having a recipe for it. 
The same is true for most other `@Rule` implementations as well as `@RunsWith` and `@Parameters`. While there are some recipes available for those I was not too happy with their results.

Reply:
I see, thanks.
I'll see what I can do. 👍